execution-env.md

```diff
@@ -570,6 +570,7 @@ Fusing and reassociation of floating point operations is allowed when those inst
 
 *   OpUndef is not allowed.
 *   OpVectorShuffle may not have a component literal with value 0xFFFFFFFF.
+*   Supported and unsupported OpCodes are iterated in [Appendix A](#a-supported-opcodes) and [Appendix B](#b-unsupported-opcodes).
```

Comment:
We should not list the _unsupported_ opcodes.  Just keep it to the whitelist.
Also, I would not make this a bullet point.  I would just pull it out as its own paragraph.

Reply:
Done